### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by marice-romero**

### CODE REVIEW

File: 01-node-tutorial/answers/globals.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+MY_VAR = "Hi there!";
```
Comment (jamisonvalentine) [inline 01-node-tutorial/answers/globals.js:1]: The instructions may not have been as clear as they could have been. Here, you actually do not need this statement: `MY_VAR = "Hi there"`.

In the assignment, there is an instruction for globals.js that reads: 
"Set an environment variable with the following command: `export MY_VAR="Hi there!"`"

This command should actually be done in your CLI or integrated terminal before running `node globals`

**Review by jamisonvalentine — CHANGES_REQUESTED** [review]
Just one requested change in `globals.js` before I can approve.

File: 01-node-tutorial/answers/fs-async.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  console.log("wrote first line");
+  writeFile("./temporary/fileB.txt", line2, { flag: "a" }, (err, result) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+    console.log("wrote second line");
+    writeFile("./temporary/fileB.txt", line3, { flag: "a" }, (err, result) => {
+      if (err) {
```
Comment (jamisonvalentine) [inline 01-node-tutorial/answers/fs-async.js:22]: You are missing the `readFile` call inside of this callback function.

File: 01-node-tutorial/answers/globals.js (hunk 1 above)
Comment (marice-romero) [reply 01-node-tutorial/answers/globals.js:1]: Got it!

File: 01-node-tutorial/answers/fs-async.js (hunk 2 above)
Comment (marice-romero) [reply 01-node-tutorial/answers/fs-async.js:22]: Sorry, I think I'm a bit confused - I thought we were supposed to just use writeFile to write three lines to a new .txt file. What file should I be using readFile with?

Comment (jamisonvalentine) [reply 01-node-tutorial/answers/fs-async.js:22]: You are right. My apologies

**Review by jamisonvalentine — APPROVED** [review]
__

File: 01-node-tutorial/answers/fs-async.js (hunk 2 above)
Comment (marice-romero) [reply 01-node-tutorial/answers/fs-async.js:22]: No worries! Thanks for checking!

**Review by jamisonvalentine — APPROVED** [review]
I thought I had already approved this, not sure what happened. I did not mean to hold you up at all. Will be much quicker with review and approval next time.

**Pull request merged**